student.rb

```diff
@@ -0,0 +1,13 @@
+require relative './person.rb'
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(classroom, name, age, can_use_services)
+    super(name, age, can_use_services)
```

Comment:
- [ ] Nice job however the `can_use_services` parameter is not correct instead you should use the **parent_permission**